notebooks/official/model-registry/bqml-vertexai-model-registry.ipynb

```diff
@@ -0,0 +1,922 @@
+{
```

Comment:
<p>Global comment, you can change "BigQuery Machine Learning" to just "BigQuery ML" since that's the official service name :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Fixed :)